app/src/main/java/com/example/musicapp/dao/MusicAppDAO.kt

```diff
@@ -3,14 +3,12 @@ package com.example.musicapp.dao
 import com.example.musicapp.dto.Instrument
 import retrofit2.Call
 import retrofit2.http.GET
-import retrofit2.http.Query
```

Comment:
Removed unused import statement as stated: https://www.jetbrains.com/help/resharper/Namespace_Imports_Style.html#removing-redundant-namespace-impoprts-and-optimizing-existing-ones